docs/camino-messenger/bot/partner-plugin.md

```diff
@@ -0,0 +1,62 @@
+---
+sidebar_position: 15
+title: Partner Plugin
+description: Camino Messenger Bot Partner Plugin
+---
+
+# Camino Messenger Bot Partner Plugin
+
+:::info DRAFT DOCUMENTATION NOTICE
+
+Please be aware that this document is currently a draft and is undergoing
+active development. Content, guidelines, and instructions may be subject to
+change.
+
+:::
+
+:::caution 🚧 ALPHA CODE NOTICE 🚧
+
+This application is in the alpha phase of development. It is important to note
+that during this stage, breaking changes may occur without advance notice.
+Users should proceed with caution.
+
+:::
+
+## What is Partner Plugin?
+
+The Camino Messenger Bot facilitates integration with existing systems through
+an application developed by the partner, whether they are a distributor or a
+provider. It comes with a sample implementation of this application to
```

Comment:
here I would further elaborate by saying something along the lines of: 

> The partner plugin acts as a middleware between the camino messenger bot and an existing system and is expected to expose a gRPC server similar to that of the bot's. Thus, each request that is not handled exclusively by the bot, is passed onto the partner plugin and upon necessary transformation gets forwarded to the distributor's/provider's existing system. 
So, if for instance an existing system offers a REST API to accept search queries, then the role of the partner plugin can be summarized as follows:
> - accept requests from the bot through a gRPC server
> - contact the existing system via REST call

Reply:
Added. 👍🏼